### DISCUSSION TITLE
Two Weeks Since Account Hijack, Still No Response!

### DISCUSSION THREAD

**Discussion by MrZhuxs** [community/community#128375] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey fellow developers, 

I'm facing a frustrating situation and wondering if anyone else has experienced something similar, and what the typical response time is for Github Support.

my main Github account was hijacked on May 28th! I immediately contacted Github Support and submitted an account recovery request, hoping to regain control.

However, to my utter disbelief, it's been a full two weeks (today is June 13th), and I haven't received a single response from Github Support! 

I've tried logging into Github Support multiple times to check on the status of my case, but it's like shouting into a void. My account holds years of code, important projects, and collaborations. It's incredibly frustrating to be locked out with zero communication.

So, Is there anything else I can do to expedite the process?

Any insights or advice would be greatly appreciated. Hopefully, Github Support will see this and address my issue soon!

**Comment by ebndev** [comment]
Hi @MrZhuxs,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
